<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupDlgClass</class>
 <widget class="QDialog" name="GroupDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupDlg</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="8">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="buttonNew">
     <property name="toolTip">
      <string>New group</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/new.png</normaloff>:/Entrance/Images/new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonDel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Delete group</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/button_cancel.png</normaloff>:/Entrance/Images/button_cancel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Edit group</string>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/Notepad2.png</normaloff>:/Entrance/Images/Notepad2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="buttonUnite">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Merge groups</string>
     </property>
     <property name="text">
      <string>Merge</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/Package_02.png</normaloff>:/Entrance/Images/Package_02.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="buttonJoin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Let current word join selected group</string>
     </property>
     <property name="text">
      <string>Join</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/User_02.png</normaloff>:/Entrance/Images/User_02.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="buttonExport">
     <property name="toolTip">
      <string>Export words</string>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/ni_png_2_0837.png</normaloff>:/Entrance/Images/ni_png_2_0837.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLineEdit" name="lineEditCurrentWord">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLineEdit" name="lineEditSearch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Keywords</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="buttonNext">
     <property name="toolTip">
      <string>Find next</string>
     </property>
     <property name="icon">
      <iconset resource="MyWords.qrc">
       <normaloff>:/Entrance/Images/rightarrow.png</normaloff>:/Entrance/Images/rightarrow.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>  Current word</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditSearch</tabstop>
  <tabstop>buttonJoin</tabstop>
  <tabstop>buttonDel</tabstop>
  <tabstop>buttonNew</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>buttonNext</tabstop>
 </tabstops>
 <resources>
  <include location="MyWords.qrc"/>
 </resources>
 <connections/>
</ui>
